src/Common/tests/System.Net/Sockets/Performance/SocketTestClient.cs

```diff
@@ -238,9 +238,23 @@ public Task<long> RunTest()
         }
 
         protected abstract string ImplementationName();
-        
-        //TODO: Either find a different fast way to compare or extract this in Interop.
```

Comment:
Why is that important in the tests?  This meaningfully affects test running time?